yoti-stub/src/data/getMediaContent/gbPassportResponse.ts

```diff
@@ -1,20 +1,20 @@
 export const GBR_PASSPORT = {
-	"full_name": "ANGELA ZOE UK SPECIMEN",
-	"date_of_birth": "1988-12-04",
+	"full_name": "Kenneth Decerqueira",
+	"date_of_birth": "1965-07-08",
 	"nationality": "GBR",
-	"given_names": "ANGELA ZOE",
-	"family_name": "UK SPECIMEN",
+	"given_names": "Kenneth",
+	"family_name": "Decerqueira",
 	"place_of_birth": "CROYDON",
 	"gender": "FEMALE",
 	"document_type": "PASSPORT",
 	"issuing_country": "GBR",
 	"document_number": "533401372",
-	"expiration_date": "2025-09-28",
+	"expiration_date": "2030-01-01",
 	"date_of_issue": "2015-09-28",
 	"issuing_authority": "HMPO",
 	"mrz": {
 			"type": 2,
-			"line1": "P<GBRUK<SPECIMEN<<ANGELA<ZOE<<<<<<<<<<<<<<<<",
+			"line1": "P<GBRUK<SPECIMEN<<Kenneth<Decerqueira<<<<<<<<<<<<<<<<",
```

Comment:
I think the MRZ data is always in caps? It may be we need to make any subsequent comparisons case insensitive?